shlr/grub/fs/minix.c

```diff
@@ -327,11 +327,7 @@ grub_minix_lookup_symlink (struct grub_minix_data *data, int ino)
 static grub_err_t
 grub_minix_find_file (struct grub_minix_data *data, const char *path)
 {
-#ifndef _MSC_VER
```

Comment:
@radare is it "right" to change the grub code this way? shouldn't we just sync the codebase with a newer version of grub or something like that?